source/_integrations/group.markdown

```diff
@@ -39,10 +39,20 @@ ha_integration_type: helper
 
 The group integration lets you combine multiple entities into a single entity. Entities that are members of a group can be controlled and monitored as a whole.
 
-This can be useful for cases where you want to control, for example, the
-multiple bulbs in a light fixture as a single light in Home Assistant.
-
-Home Assistant can group multiple binary sensors, covers, events, fans, lights, locks, media players, switches as a single entity, with the option of hiding the individual member entities.
+This can be useful for cases where you want to control, for example, the multiple bulbs in a light fixture as a single light in Home Assistant. You also have the option of hiding the individual member entities in a group.
```

Comment:
```suggestion
This can be useful, for example, in cases where you want to control multiple bulbs in a light fixture as a single light in Home Assistant. You also have the option of hiding the individual member entities in a group.
```